test/k8sdeployment/deployment_test.go

```diff
@@ -78,32 +78,34 @@ func TestKubernetesDeploymentStarts(t *testing.T) {
 
 func TestKubernetesDeploymentHealthy(t *testing.T) {
```

Comment:
Since we changed what "health" means in CoreDNS, this test is less useful.

_In another PR_: We should add a test of Ready... perhaps adding a new test, or renaming this to `TestKubernetesDeploymentHealthyAndReady`: and have it also check the ready endpoint.

Reply:
Yes, I'll open another PR to add a test for Ready